spec/models/notification_policy_spec.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
The `summarize!` method in this class is interesting enough that it should have coverage.